src/Sarif.Driver/Sarif.Driver.csproj

```diff
@@ -28,8 +28,8 @@
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), build.props))\build.props" />
   <ItemGroup>
     <Reference Include="CommandLine, Version=2.0.275.0, Culture=neutral, PublicKeyToken=de6f01bd326f8c32, processorArchitecture=MSIL">
-      <HintPath>..\packages\CommandLineParser.2.0.275-beta\lib\net45\CommandLine.dll</HintPath>
-      <Private>True</Private>
+      <SpecificVersion>False</SpecificVersion>
+      <HintPath>..\..\references\CommandLine.dll</HintPath>
```

Comment:
I'm just curious why the reference in the csproj file had to change. Can't the csproj file and the nuspec both include the CommandLine.dll from the package?